docs/Examples.md

```diff
@@ -426,8 +426,11 @@ spec:
   defaultsUrl: /mnt/defaults/default.yml
 ```
 
-`volumes` will mount the ConfigMap in all of your pods under the
-`/mnt/licenses` directory.
+`volumes` config volume mounts the `splunk-defaults` ConfigMap
+(therefore the `default.yml` file) on all the Splunk Enterprise
+pod/s of the Custom Resource(Standalone) under the `/mnt/defaults`
+directory. `defaultsUrl` represents the full path to the
+`default.yml` configuration file on the pod/s.
 
```

Comment:
In the above example, `volumes` will mount the `splunk-defaults` ConfigMap with `default.yml` file under the `/mnt/defaults` directory on all pods of the Custom Resource `Standalone`.

`defaultsUrl` represents the full path to the `default.yml` configuration file on the pods. In addition, `defaultsUrl` may specify one or more local paths or URLs, each separated by a comma. For example, you can use a `generic.yml` with common	by a comma

Reply:
Updated